app/src/theme.ts

```diff
@@ -95,6 +95,7 @@ export const buildNotdTheme = (): ITheme => {
         // 'border-color': 'rgba(255, 255, 255, 0.5)',
         // 'border-style': 'solid',
         margin: '0',
+        'border-radius': '0',
```

Comment:
this seems wrong - you're changing all cards on the ui everywhere to have no radius?!